packages/xchain-polkadot/src/client.ts

```diff
@@ -30,6 +30,8 @@ export interface PolkadotClient {
   getSS58Format(): number
   getWsEndpoint(): string
   estimateFees(params: TxParams): Promise<Fees>
+
+  purgeProvider(): void
```

Comment:
^ `purgeProvider` can be removed from interface. Since `getAPI` is private we can make `purgeProvider` as well.